plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestTrinoRESTCatalogConnectorTest.java

```diff
@@ -0,0 +1,170 @@
+                        .add(LINE_ITEM)
+                        .build())
+                .build();
+    }
+
+    @Override
+    @Test
+    public void testCreateSchemaWithLongName()
+    {
+        throw new SkipException("We're hitting max URI length before hitting the limit of the backing catalog");
+    }
+
+    @Override
+    @Test
+    public void testCreateTableWithLongTableName()
+    {
+        throw new SkipException("We're hitting max URI length before hitting the limit of the backing catalog");
+    }
+
+    @Override
+    @Test
+    public void testRenameTableToLongTableName()
+    {
+        throw new SkipException("We're hitting max URI length before hitting the limit of the backing catalog");
+    }
+
+    @Override
+    @Test
+    public void testAddColumnConcurrently()
+    {
+        throw new SkipException("Concurrent commit not supported");
+    }
+
+    @Override
+    @Test
+    public void testCreateTableAsSelectSchemaNotFound()
+    {
+        throw new SkipException("Backing catalog creates schema automatically.");
+    }
+
+    @Override
+    @Test
+    public void testCreateTableSchemaNotFound()
+    {
+        throw new SkipException("Backing catalog creates schema automatically.");
+    }
+
+    @Override
+    @Test
+    public void testShowCreateSchema()
+    {
+        // Overridden due to REST Catalog not supporting namespace principal
+        assertThat(computeActual("SHOW CREATE SCHEMA tpch").getOnlyValue().toString())
+                .matches("CREATE SCHEMA iceberg.tpch\n" +
+                        "WITH \\(\n" +
+                        "\\s+location = '.*/iceberg_data/tpch'\n" +
+                        "\\)");
+    }
+
+    @Override
+    @Test
+    public void testShowCreateTable()
+    {
+        File tempDir = getDistributedQueryRunner().getCoordinator().getBaseDataDir().toFile();
+        assertThat((String) computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .matches("\\QCREATE TABLE iceberg.tpch.orders (\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar,\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar,\n" +
+                        "   clerk varchar,\n" +
+                        "   shippriority integer,\n" +
+                        "   comment varchar\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = '" + PARQUET.name() + "',\n" +
+                        "   format_version = 2,\n" +
+                        "   location = '" + tempDir + "/iceberg_data/tpch/orders'\n" +
+                        ")");
+    }
+
+    @Override
+    @Test
+    public void testTableNameCollision()
+    {
+        throw new SkipException("Backing catalog supports writing to existing locations");
```

Comment:
Please use `assertThatThrownBy` so that we can catch the future change. Other places are also same. Generally, we avoid using  `SkipException` as much as possible.

Reply:
updated to `assertThatThrownBy` for this particular test method. The other test methods still use `SkipException` because the test expects an exception to be thrown, but the underlying backend catalog doesn't throw one, so using `SkipException` makes more sense in those cases